tcms/testplans/templates/testplans/get.html

```diff
@@ -110,26 +112,51 @@ <h2 class="card-pf-title">
 <template id="test_case_row">
     <div class="list-group-item">
         <div class="list-group-item-header">
-            <div class="list-view-pf-expand">
-                <span class="fa fa-angle-right"></span>
-            </div>
-            <div class="list-view-pf-checkbox">
-                <input type="checkbox">
-            </div>
-            <div class="list-view-pf-actions">
-                <div class="dropdown pull-right dropdown-kebab-pf">
-                    <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                        <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
-                        <li><a href="#">{% trans 'Change status' %}</a></li>
-                    </ul>
-                </div>
-            </div>
             <div class="list-view-pf-main-info">
-                <!-- todo: this text is slightly misaligned. for some reason goes to the top -->
-                <a class="testcase" href="#"></a>
+                <div class="list-view-pf-left">
+                    <span class="fa fa-angle-right"></span>
+                </div>
+                <div class="list-view-pf-checkbox">
+                    <input type="checkbox">
+                </div>
+                <div class="list-view-pf-body">
+                    <div class="list-view-pf-actions">
```

Comment:
`style="margin-top: 0; margin-bottom: 0"`  because of this same extra top/bottom margin which make the row bigger.

Also with the next changes described below you should place pf-actions above the pf-body element.

Reply:
Here I needed margin-left because otherwise the menu is not shown

Reply:
> Also with the next changes described below you should place pf-actions above the pf-body element.

If you place the entire list-view-pf-actions element outside of list-view-pf-body then you will not need the margin-left style change. From what I can tell this is how it is done in the examples. I'm looking at https://www.patternfly.org/v3/pattern-library/content-views/tree-list-view/tree-list-view-single-child-expansion.html